### PR TITLE
[WTF-809] Add documentation on data view conditional editability [10.7.0]

### PR DESCRIPTION
**Pull request by mlaponder**
This doc changes describe the new feature to be introduced in Mendix 10.7.0 allowing to specify conditional editability on data view level.

### CODE REVIEW

File: content/en/docs/refguide/modeling/pages/common-widget-properties.md [hunk 1]
```diff
@@ -151,12 +151,12 @@ The attribute can be of one of the following [data types](/refguide/data-types/)
...
+| Default     | The value is editable the containing data container is editable and if security allows it such as if the user that is signed in has write access to the selected attribute (default value for widgets outside a snippet). |
+| Inherited from snippet call | Set to **Default** or **Never** by the containing data container of the snippet call (default value for widgets inside a snippet).                                                                                        |
+| Never       | The value is never editable.                                                                                                                                                                                              |
+| Conditionally | The value is editable if the specified condition holds (see below).                                                                                                                                                       |
```
Comment (ancajuravle) [inline content/en/docs/refguide/modeling/pages/common-widget-properties.md]: @mlaponder we can now also have the option "Conditionally (combined with inherited condition)", right?

File: content/en/docs/refguide/modeling/pages/common-widget-properties.md [hunk 2]
```diff
@@ -151,12 +151,12 @@ The attribute can be of one of the following [data types](/refguide/data-types/)
...
-| Inherited from snippet call | Set to **Default** or **Never** by the containing data container of the snippet call (default value for widgets inside a snippet). |
-| Never       | The value is never editable.                                 |
-| Conditionally | The value is editable if the specified condition holds (see below). |
+| Value       | Description                                                                                                                                                                                                               |
+| ----------- |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Default     | The value is editable the containing data container is editable and if security allows it such as if the user that is signed in has write access to the selected attribute (default value for widgets outside a snippet). |
```
Comment (ancajuravle) [inline content/en/docs/refguide/modeling/pages/common-widget-properties.md]: Missing an "if",  "The value is editable **if** the containing...".

Comment (mlaponder) [reply content/en/docs/refguide/modeling/pages/common-widget-properties.md]: Nice catch, fixed.

File: content/en/docs/refguide/modeling/pages/common-widget-properties.md (hunk 1 above)
Comment (mlaponder) [reply content/en/docs/refguide/modeling/pages/common-widget-properties.md]: You are right, fixed

**Review by ancajuravle — APPROVED** [review]

File: content/en/docs/refguide/modeling/pages/common-widget-properties.md [hunk 3]
```diff
@@ -151,12 +151,13 @@ The attribute can be of one of the following [data types](/refguide/data-types/)
...
+| Inherited from snippet call                       | Set to **Default** or **Never** by the containing data container of the snippet call (default value for widgets inside a snippet).                                                                                           |
+| Never                                             | The value is never editable.                                                                                                                                                                                                 |
+| Conditionally                                     | The value is editable if the specified condition holds (see below).                                                                                                                                                          |
+| Conditionally (combined with inherited condition) | The value is editable if the specified condition holds and the conditionas for all conditional editable data containers also holds.                                                                                          |
```
Comment (ancajuravle) [inline content/en/docs/refguide/modeling/pages/common-widget-properties.md]: `the conditionas for all conditional editable data containers also holds` 
@mlaponder there's a typo still "conditionas" and if it's about the condition**s**, should it be "hold"?

Comment (mlaponder) [reply content/en/docs/refguide/modeling/pages/common-widget-properties.md]: Oops 🤦 and you are right. I fixed and amended my last commit.

**Review by ancajuravle — APPROVED** [review]

**Comment by mlaponder** [comment]
@ConnorLand This PR has been reviewed and approved by our team, it is up to you now

**Review by ConnorLand — APPROVED** [review]

File: content/en/docs/refguide/modeling/pages/data-widgets/data-view.md [hunk 4]
```diff
@@ -88,13 +88,32 @@ The data view supports the following types of data source: context, microflow, n
...
+
+#### 4.3.2.1 Based on Attribute Value
+
+When selected, this enables the widgets in the data view when a particular attribute has a certain value. Only Boolean and enumeration attributes can be used for this purpose.
+
+#### 4.3.2.2 Based on Expression
+
+When selected, this enables the widget when a provided [expression](/refguide/expressions/) evaluates to true. The object of the data view is available inside an expression as the `$currentObject` variable.
+
+The expression provided is evaluated in the browser. Currently it does not support all the functions that are available in microflows. The autocomplete function will only list those functions which are supported.
```
Comment (ConnorLand) [inline content/en/docs/refguide/modeling/pages/data-widgets/data-view.md:114]: Is this discrepancy with microflow functionality going to be the case for a long time? If yes, then it's safe to keep this language in the doc.

Comment (ancajuravle) [reply content/en/docs/refguide/modeling/pages/data-widgets/data-view.md:114]: Yes, that's gonna be the case Connor. It will probably never change even.

**Pull request merged**